api/docs/release.dox

```diff
@@ -241,6 +241,9 @@ Further non-compatibility-affecting changes include:
    trace analysis tool framework.
  - Added "-t drmemtrace" as the preferred launcher for the drmemtrace/drcachesim
    trace analysis tool framework.
+ - Added func_marker_filter_t filter to #dynamorio::drmemtrace::record_filter_t to filter
+   #dynamorio::drmemtrace::TRACE_MARKER_TYPE_FUNC_ID (and _ARG, _RETVAL, _RETADDR) entries
+   based on the marker value (i.e., the function ID).
```

Comment:
Mentioning all the marker types and then saying it filters based on marker value seems confusing to me: the values of the other markers are different.  I would say it filters function markers based on the "function id".

Reply:
Fixed.